### PR TITLE
archlinux packaging & required fixes

### PR DESCRIPTION
**Pull request by shindo**
You can read about initializer list problem here:
http://stackoverflow.com/questions/26947704/implicit-conversion-failure-from-initializer-list

I'm not expect you to merge this pull request. You can just take a look at tests' fixes and apply them if necessary.

### CODE REVIEW

File: archlinux/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Maintainer: Danil Osherov <shindo@yandex-team.ru>
+
+pkgname=blackhole
+pkgdesc="Yet another C++ logging library."
+pkgver=0.2.3
+pkgrel=1
+arch=('x86_64')
+url=('https://github.com/3Hren/blackhole')
+license=('LGPL3')
```
Comment (3Hren) [inline archlinux/PKGBUILD]: It's MIT actually.

File: archlinux/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+depends=('boost>=1.46')
+makedepends=('git' 'cmake')
+source=("$pkgname"::"git://github.com/shindo/blackhole.git#tag=v$pkgver")
```
Comment (3Hren) [inline archlinux/PKGBUILD]: I think this line conflicts with the `url` one (check user prefix).

**Comment by 3Hren** [comment]
Thanks for this PR, but I can merge it only after Travis accepts it. Please see what's going wrong there.

File: archlinux/PKGBUILD (hunk 1 above)
Comment (shindo) [inline archlinux/PKGBUILD]: I used PKGBUILD from some other project as a base for this one. So, sure, I will change it.

File: archlinux/PKGBUILD (hunk 2 above)
Comment (shindo) [inline archlinux/PKGBUILD]: Nope, it was on purpose. I use git tags to checkout source code for specific version (format is just like yours - "v(version)", e.g. v0.2.3).
It wasn't possible to add tag to your repository (I haven't necessary permissions), so I used mine. 

I'll change it back to yours repository, just don't forget to update v0.2.3 tag.

**Pull request merged**